cg/apps/lims/api.py

```diff
@@ -381,3 +376,12 @@ def _find_twist_capture_kits(artifacts, udf_key):
             if artifact.udf.get(udf_key) is not None
         )
         return capture_kits
+
+    def get_sample_comment(self, sample_id: str):
```

Comment:
Could you add return types for these? like:
```suggestion
    def get_sample_comment(self, sample_id: str) -> str:
```
Or whatever it is returning